llvm/test/ExecutionEngine/JITLink/AArch64/ELF_relocations.s

```diff
@@ -51,6 +51,21 @@ test_adr_prel_lo21:
 ## to test this bit better
 adr_data = test_adr_prel_lo21 + 0xe46f2
 
+# Check R_AARCH64_LD_PREL_LO19 relocation of a local symbol
```

Comment:
@eymay Thank you very much for the review and idea to simplify the test. My idea was to show how addends in the relocation will be handled by the linker, this is important because the previous implementation just asserted the addend to zero what is not always true, I saw really generated code where the `R_AARCH64_LD_PREL_LO19` relocation had a non-zero addend. But I overengineering the code with nops to give the loaded symbol a place to be, your version is much simpler and also works.